Curs/src/style/login.less

```diff
@@ -221,34 +352,105 @@ a {
 @media screen and (max-width: 1200px) and (min-width: 992px) {
     /* Styles */
 
-    .footer-section {
-        text-align: center;
+    #alert {
+        color: red;
+        font-size: 20px;
+        width: 97%;
     }
 
     .feedback-image {
-        float: left !important;
+        width: 43%;
+        margin-left: 325px;
+        height: 300px;
+    }
+
+    .right-text-description {
+        width: 100%;
+        margin-top: 350px;
+        text-align: center;
+        float: none;
+    }
+
+    .left-text-description {
+        width: 100%;
+        margin-top: 100px;
+        text-align: center;
+        float: left;
+        padding-left: 0;
     }
 
+    .text-description-2 {
```

Comment:
Any particular reason why we're repeating this? It looks like L322-L324 (among with a couple of other items) are exact duplicates.

Reply:
I also changed the name of the class there. :)